src/RageSurfaceUtils_Palettize.cpp

```diff
@@ -305,7 +305,7 @@ void RageSurfaceUtils::Palettize( RageSurface *&pImg, int iColors, bool bDither
 		}
 	}
 
-	delete acolormap;
+	free(acolormap);
```

Comment:
Can we instead allocate acolormap using new?

new/delete is preferred over malloc/free (although ideally we wouldn't use either and use smart pointer but that we can tackle that another time).